### ISSUE TITLE
problem in the installation of NuSMV-2.6.0 in #make

### ISSUE THREAD

**Issue by crptcusco** [hklarner/SWP2016#20]
I was installing the NuSMV-2.6.0 program and after executing the `make` command, these error lines appear, attached file.

`Building C object build-cudd / util / CMakeFiles / CUDD_UTIL_LIB.dir / pipefork.c.o`
[isue NUSMV.txt](https://github.com/hklarner/NuSMV-a/files/3124213/isue.NUSMV.txt)

My operating system is: Ubuntu 18.04 64bits
Already execute these commands, I think you can add to the installation manual:
```
sudo apt-get install flex
sudo apt-get install bison
```
Thanks

**Comment by hklarner** [comment]
Yes, you are right. The installation of `NuSMV-a` is the same as the installation of `NuSMV-2.6.0` which requires `flex` and `bison`.

**Comment by crptcusco** [comment]
Yes but , when i try to install, with the comand `#make  `, say that have one error , i attached the error in this file. 
[isue.NUSMV.txt](https://github.com/hklarner/NuSMV-a/files/3124420/isue.NUSMV.txt)

**Comment by hklarner** [comment]
Now, I am facing the same problem as you. Did you eventually succeed in compiling NuSMV-a?
